### DISCUSSION TITLE
Debian Buster support

### DISCUSSION THREAD

**Discussion by tigattack** [trapexit/mergerfs#1112] [Q&A] [answered]
As of the 2.34.0 release, I see there are no assets for Debian Buster (10). Is there no longer support for Buster?

P.S. thanks to all contributors for their work on this great project!

**Comment by trapexit** [comment]
Nothing is explicitly not supported (at least regarding Linux). As you can see there are a ton of packages built and sometimes they fail because of spurious errors or container images change from under me. I just upload what it spits out and don't really have a good setup to check for errors. I'll take a look to see what happened.

**Reply by tigattack** [reply]
Understood, thank you for resolving!

**Comment by trapexit** [comment, marked as answer]
Updated
